### ISSUE TITLE
manual settle not working because bug in mqlight-forked-amqp10

### ISSUE THREAD

**Issue by mmaestri** [mqlight/nodejs-mqlight#12]
Hi @dnwe 

There is a problem with the module **mqlight-forked-amqp10** that was inherited from original module **amqp10**. It does not allow to manually settle messages, hardcoding the auto-settle option. 

I've found the bug in **mqlight-forked-amqp10/lib/receiver_link.js** line **195**

It says

```
194  // respect settle mode in policy
195  if (this.policy.settlement === constants.settlement.manual) {
196    this.accept(message);
197  }
```

Actually it should be

```
194  // respect settle mode in policy
195  if (this.policy.settlement === constants.settlement.auto) {
196    this.accept(message);
197  }
```

As you may know, I cannot make a pull request to the **amqp10** module because the Travis CI build failed ([link](https://github.com/noodlefrenzy/node-amqp10/pull/317))

I've made a comment on the commit that failed the build, but what I really need is nodejs-mqlight working properly. Would you please correct it and make the change in **mqlight-forked-amqp10**?

Thanks

**Comment by dnwe** [comment]
@mmaestri thanks! Should be fixed by mqlight@2.0.2017062000 – please confirm once you have had a chance to test it.

**Comment by dnwe** [comment]
Closing issue, assumed fixed.
